app.js

```diff
@@ -0,0 +1,32 @@
+const articleList = document.getElementById("article-list");
+const articleTemplate = (article) => {
+  return `<div class="article">
+<p class="keyword">${article.keyword}</p>
+<h3>${article.title}</h3>
+<p class="article-content">
+  ${article.summary}</p>
+</div>`;
+};
+
+let articleData = "";
+
+const data = [
+  {
+    keyword: "hands on",
+    title: "Criando um projeto full stack com HTML, CSS e API Python",
+    summary:
+      "Montes, lacus, laoreet commodo egestas amet eget laoreet. Amet et",
+  },
+  {
+    keyword: "Heads Off",
+    title: "Live",
+    summary: "lorem lorem lorem lorem",
+  },
+];
+
+for (const article of data) {
```

Comment:
Poderíamos fazer isso no ES6+ way?
`articleList.forEach(...)`

Reply:
for-of is a new loop in ES6 that replaces both for-in and forEach() and supports the new iteration protocol.

Reply:
@torquato-nathan 
Aí passa ser uma questão de gosto entre funcional e procedural programming. Particularmente, for-of me parece mais claro e acho melhor para iniciantes.

Reply:
Particularmente vejo mais o padrão de forEach na maioria dos artigos, for... of vejo mais quando se quer usar um `await`.
Mas ok.